libraries/ms-common/src/x86/gpio_it.c

```diff
@@ -65,3 +65,7 @@ StatusCode gpio_it_trigger_interrupt(const GPIOAddress *address) {
 
   return x86_interrupt_trigger(s_gpio_it_interrupts[address->pin].interrupt_id);
 }
+
+StatusCode gpio_it_mask_interrupt(const GPIOAddress *address, bool masked) {
+  return STATUS_CODE_OK;
```

Comment:
This should probably return not implemented.

Reply:
+1 also optionally wrap in `status_code()` to add to the global status register.

Reply:
changed it to:
return status_code(STATUS_CODE_UNIMPLEMENTED);